src/scss/includes/search_form.scss

```diff
@@ -198,6 +205,10 @@ input[type="search"] {
       color: $secondary_text;
       font-size: 14px;
     }
+
+    &:first-letter {
+      text-transform: capitalize;
+    }
```

Comment:
I understand why we need this rule for intention suggestions.  
But for other types of suggestions, don't you think that the API response should be the source of truth for capitalization of place names, store brands, etc ?

Reply:
Ok, I'll put it only on categories and suggestions then.